lib/core/templates/error-content/error-content.component.stories.ts

```diff
@@ -29,120 +29,49 @@ export default {
         moduleMetadata({
             imports: [CoreStoryModule, TemplateModule],
             providers: [
-                {
-                    provide: ActivatedRoute,
-                    useValue: { params: of({}) }
-                }
+                { provide: ActivatedRoute, useValue: { params: of({}) }}
             ]
         })
     ],
     argTypes: {
         errorCode: {
-            type: 'string',
-            description: 'Component level Error Code',
+            control: 'text',
+            description: 'Error code associated with this error.',
+            defaultValue: 'UNKNOWN',
             table: {
-                category: 'Component Inputs',
```

Comment:
@marcol13  please check it with @DudaRobert

Reply:
Resolved, we're keeping this version